src/app/documentation/demos/button-group/button-group.demo.html

```diff
@@ -227,46 +229,17 @@ <h6 class="clrweb-DoxMedia-do-dont">Don't</h6>
                         </div>
                     </div>
                     <div class="clrweb-DoxMedia-text">
-                        <p>
-                            Flat buttons have no minimum width.
-                            There is a 24px margin between them.
-                        </p>
-                    </div>
-                </div>
-            </div>
-
-            <div class="col-sm-12 col-lg-4">
-                <div class="clrweb-DoxMedia is-dont-block">
-                    <div class="clrweb-DoxMedia-block">
-                        <div class="clrweb-DoxMedia-img">
-                            <div class="btn-group btn-primary">
-                                <button class="btn" style="min-width: 0">
-                                    <clr-icon shape="plus"></clr-icon>
-                                    <span class="clr-icon-title">Add</span>
-                                </button>
-                                <button class="btn" style="min-width: 0">
-                                    <clr-icon shape="favorite"></clr-icon>
-                                    <span class="clr-icon-title">Favorite</span>
-                                </button>
-                                <button class="btn" style="min-width: 0">
-                                    <clr-icon shape="download"></clr-icon>
-                                    <span class="clr-icon-title">Download</span>
-                                </button>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="clrweb-DoxMedia-text">
                         <h6 class="clrweb-DoxMedia-do-dont">Don't</h6>
                         <p>
-                            Make solid or outline buttons smaller than
-                            72px wide.
+                            Use small icon buttons in most cases. They are difficult to see and distinguish what
```

Comment:
So should we be using the small icon buttons? The first sentence here doesn't align with the two after it.